packages/data-models/skin.model.ts

```diff
@@ -21,4 +21,5 @@ import { IAppConfigOverride } from "./appConfig";
 export interface IAppSkin {
   name: string;
   appConfig?: IAppConfigOverride;
+  availableThemes?: string[];
```

Comment:
nit(blocking)
the appConfig already has theme properties so is there any real reason to elevate a level higher here? feels a bit confusing to me